packages/react-aria-components/src/ColorWheel.tsx

```diff
@@ -0,0 +1,105 @@
+import {AriaColorWheelOptions, useColorWheel} from '@react-aria/color';
+import {ColorWheelState, useColorWheelState} from '@react-stately/color';
+import {ContextValue, Provider, RenderProps, SlotProps, StyleRenderProps, useContextProps, useRenderProps} from './utils';
+import {filterDOMProps} from '@react-aria/utils';
+import {InternalColorThumbContext} from './ColorThumb';
+import React, {createContext, ForwardedRef, forwardRef, HTMLAttributes, useContext, useRef} from 'react';
+
+export interface ColorWheelRenderProps {
+  /**
+   * Whether the color wheel is disabled.
+   * @selector [data-disabled]
+   */
+  isDisabled: boolean,
+  /**
+   * State of the color color wheel.
+   */
+  state: ColorWheelState
+}
+
+export interface ColorWheelProps extends AriaColorWheelOptions, RenderProps<ColorWheelRenderProps>, SlotProps {}
+
+export const ColorWheelContext = createContext<ContextValue<Partial<ColorWheelProps>, HTMLDivElement>>(null);
+export const ColorWheelStateContext = createContext<ColorWheelState | null>(null);
+
+function ColorWheel(props: ColorWheelProps, ref: ForwardedRef<HTMLDivElement>) {
+  [props, ref] = useContextProps(props, ref, ColorWheelContext);
+  let state = useColorWheelState(props);
+  let inputRef = useRef(null);
+  let {trackProps, inputProps, thumbProps} = useColorWheel(props, state, inputRef);
+
+  let renderProps = useRenderProps({
+    ...props,
+    values: {
+      state,
+      isDisabled: props.isDisabled || false
+    },
+    defaultClassName: 'react-aria-ColorWheel',
+    defaultStyle: {
+      position: 'relative',
+      width: props.outerRadius * 2,
+      height: props.outerRadius * 2
+    }
+  });
+
+  let DOMProps = filterDOMProps(props);
+  delete DOMProps.id;
+
+  return (
+    <div
+      {...DOMProps}
+      {...renderProps}
+      ref={ref}
+      slot={props.slot || undefined}
+      data-disabled={props.isDisabled || undefined}>
+      <Provider
+        values={[
+          [ColorWheelStateContext, state],
+          [ColorWheelTrackContext, trackProps],
+          [InternalColorThumbContext, {state, thumbProps, inputXRef: inputRef, xInputProps: inputProps, isDisabled: props.isDisabled}]
+        ]}>
+        {renderProps.children}
+      </Provider>
+    </div>
+  );
+}
+
+/**
+ * A color wheel allows users to adjust the hue of an HSL or HSB color value on a circular track.
+ */
+const _ColorWheel = forwardRef(ColorWheel);
+export {_ColorWheel as ColorWheel};
+
+export interface ColorWheelTrackRenderProps extends ColorWheelRenderProps {}
+export interface ColorWheelTrackProps extends StyleRenderProps<ColorWheelTrackRenderProps> {}
+interface ColorWheelTrackContextValue extends Omit<HTMLAttributes<HTMLElement>, 'children' | 'className' | 'style'>, ColorWheelTrackProps {}
+
+export const ColorWheelTrackContext = createContext<ContextValue<ColorWheelTrackContextValue, HTMLDivElement>>(null);
+
+function ColorWheelTrack(props: ColorWheelTrackProps, ref: ForwardedRef<HTMLDivElement>) {
```

Comment:
Ended up creating a separate component for the ColorWheelTrack. I wanted the track to just be rendered as the background of the ColorWheel itself, but the clip-path we use makes this complicated. The thumb shouldn't actually be clipped, so the only way to do that is to make a separate sibling element. Couldn't reuse SliderTrack here because that expects a SliderState and exposes a few other render props that are specific to sliders.